Source/WebKit/Shared/WebPreferencesDefaultValues.cpp

```diff
@@ -245,7 +245,9 @@ bool defaultGamepadVibrationActuatorEnabled()
 bool defaultDigitalCredentialsEnabled()
 {
 #if HAVE(DIGITAL_CREDENTIALS_UI)
-    return true;
+    Ref webProcess = WebProcess::singleton();
+    bool enabled = webProcess->parentProcessHasEntitlement("com.apple.developer.web-browser"_s) || webProcess->parentProcessHasEntitlement("com.apple.developer.identity-document-services.web-presentment-controller"_s);
+    return enabled;
```

Comment:
Looking at the other code in this file I suspect you want and maybe you should also use `dispatch_once`?
```suggestion
    static bool hasEntitlement = webProcess->parentProcessHasEntitlement("com.apple.developer.web-browser"_s) || webProcess->parentProcessHasEntitlement("com.apple.developer.identity-document-services.web-presentment-controller"_s);
    return hasEntitlement;
```

Reply:
Moved to using dispatch_once